app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,19 @@
+from abc import ABC, abstractmethod
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+    def __init__(self, nickname, favourite_dish):
+        super(Dwarf, self).__init__(nickname)
+        self._favourite_dish = favourite_dish
+
+    def eat_favourite_dish(self):
+        print(f"{self.nickname} is eating {self._favourite_dish}")
+
+    @abstractmethod
+    def get_rating(self):
+        pass
+
+    @abstractmethod
+    def player_info(self):
+        pass

```

Comment:
Maybe it's a little chit but i don't wrote the last two functions with `@anstractmethod`. I define them in child classes and everithing works)

Reply:
If you have an abstract class that inherits another abstract class you don't need to repeat abstract methods from parent.